### PR TITLE
Simplifying Service APIs Meeting Times

### PR DESCRIPTION
**Pull request by robscott**
Now that we've launched v1alpha1, we've decided to slow down our meeting frequency to once a week. The results of https://strawpoll.com/w5p5peyq7 have shown that Wednesdays at 11am Pacific seems to work best for most maintainers. We've already updated the sig-network calendar and are in the process of updating our own community page: https://github.com/kubernetes-sigs/service-apis/pull/479

/sig network
/cc @bowei 
/assign @thockin

### CODE REVIEW

File: sig-network/README.md [hunk 1]
```diff
@@ -18,11 +18,7 @@ The [charter](charter.md) defines the scope and governance of the Network Specia
...
-* Service APIs Meeting (APAC Friendly Time): [Thursdays at 16:30 PT (Pacific Time)](https://zoom.us/j/441530404) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:30&tz=PT%20%28Pacific%20Time%29).
-  * [Meeting notes and Agenda](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit).
-* Service APIs Meeting (EMEA Friendly Time): [Thursdays at 10:30 PT (Pacific Time)](https://zoom.us/j/140577552) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:30&tz=PT%20%28Pacific%20Time%29).
-  * [Meeting notes and Agenda](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit).
-* Service APIs Office Hours: [Wednesdays at 15:00 PT (Pacific Time)](https://zoom.us/j/289465529) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=PT%20%28Pacific%20Time%29).
+* Service APIs Meeting: [Wednesdays at 11:00 PT (Pacific Time)](https://zoom.us/j/441530404) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
```
Comment (bowei) [inline sig-network/README.md:21]: biweekly?

Comment (robscott) [reply sig-network/README.md]: Woops, fixed now.

**Comment by mrbobbytables** [comment]
/approve
/hold
for @bowei or @thockin sign off 👍

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**